cmd/head/head.go

```diff
@@ -0,0 +1,43 @@
+package main
+
+import (
+	"bufio"
+	"flag"
+	"fmt"
+	"os"
+	"path/filepath"
+	"log"
+)
+func check(e error) {
+	if e != nil {
+		log.Fatal(e)
+	}
+}
+
+var(
+	numberOfLines int
+)
+
+func main() {
+	headCmd := flag.NewFlagSet("head", flag.ExitOnError)
```

Comment:
you can use `flag.BoolVar` or `flag.IntVar` directly instead of new flag set for each command, check [here](https://github.com/codescalersinternships/Coreutils-Nabila-Sherif/blob/b3420dbd184d5a268d72353cb96f32a29b3075cb/cmd/head/main.go#L18) it's much easier